domain/user.go

```diff
@@ -134,7 +134,7 @@ func IsValidAccountURL(accountType AccountType, URL string) bool {
 		PIXIV:      regexp.MustCompile(`^https://www\.pixiv\.net/users/[0-9]+`),
 		GITHUB:     regexp.MustCompile(`^https://github\.com/[a-zA-Z0-9-]+$`),
 		QIITA:      regexp.MustCompile(`^https://qiita\.com/[a-zA-Z0-9-_]+$`),
-		ZENN:       regexp.MustCompile(`^https://zenn\.dev/[a-zA-Z0-9.]+$`),
+		ZENN:       regexp.MustCompile(`^https://zenn\.dev/[a-zA-Z0-9.-_]+$`),
```

Comment:
```suggestion
		ZENN:       regexp.MustCompile(`^https://zenn\.dev/[a-z0-9._]+$`),
```

<img width="500" alt="image" src="https://github.com/traPtitech/traPortfolio/assets/18237819/96f8d721-690a-4c89-9eb9-545c1bf1a5ee">

Reply:
.も消して欲しいです！

Reply:
あ、ほんとだ！ `.` 消しました〜

Reply:
感謝 スマホフレンドリー